testenv/testenv_arm32.properties

```diff
@@ -9,6 +9,5 @@ OPENJ9_SYSTEMTEST_BRANCH=openj9-0.36.0
 ADOPTOPENJDK_SYSTEMTEST_REPO=https://github.com/adoptium/aqa-systemtest.git
 ADOPTOPENJDK_SYSTEMTEST_BRANCH=v0.9.6
 JDK8_REPO=https://github.com/adoptium/aarch32-jdk8u.git
-JDK8_BRANCH=dev
+JDK8_BRANCH=jdk8u362-ga-aarch32-20230119
```

Comment:
Needs to be: jdk8u362-b09-aarch32-20230119

Reply:
Fixed